pallets/parachain-staking/src/lib.rs

```diff
@@ -448,22 +449,16 @@ pub mod pallet {
 					selected_collators_number: collator_count,
 					total_balance: total_staked,
 				});
-				weight = weight.saturating_add(T::WeightInfo::round_transition_on_initialize(
-					collator_count,
-					delegation_count,
-				));
+				// account for Round and Staked writes
+				weight = weight.saturating_add(T::DbWeight::get().reads_writes(0, 2));
```

Comment:
I'm not feeling super comfortable with manually adding reads/writes here. Especially since it's not trivial to know storage accesses (and changes to them in course of time). Is there no other way that the benchmarks themself record this info?

Reply:
They would if we benchmarked `on_initialize` as a whole again like we were doing before this change. Even then we still have to add some manually-accounted-for weight for `on_finalize` here.

It might be a good sanity check to continue benchmarking `on_initialize` but only for the purposes of comparing its db accesses to our own... but this doesn't solve the disconnect between changing any logic that affects `on_initialize` and updating these manual db op weights.

By the way, there are numerous examples in Substrate where db reads/writes are done manually like this.